.github/workflows/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# Info about CODEOWNERS
+# https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+# Global owners - the current maintainers
+# https://libra.org/en-US/blog/lip-maintainers/
+*   @dahliamalkhi @kphfb @sblackshear @davidiw @aching
```

Comment:
We can even make this more granular by having a code owner(s) per LIP as well, given that LIPs are in their own directory. For example:

```
/lips/lip-1.mdx     @kphfb 
```